### PR TITLE
Add prim ops neg.Scalar

### PR DESCRIPTION
**Pull request by larryliu0820**
Summary: This is required from Llava preprocess. The graph contains this node:

```python
%neg : [num_users=1] = call_function[target=operator.neg](args = (%floordiv_2,), kwargs = {})
```

This is from this line:

```python
r_pad = -((target_w - img.shape[2]) // -2)
```

Test Plan: See the model is being exported to .pte file

Reviewers:

Subscribers:

Tasks:

Tags:

### CODE REVIEW

**Review by lucylq — APPROVED** [review]

**Review by lucylq — COMMENTED** [review]
Add to the tests in kernels/prim_ops/test/ ?

File: kernels/prim_ops/test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+
+include(${EXECUTORCH_ROOT}/build/Test.cmake)
+
+set(_test_srcs prim_ops_test.cpp)
+
+et_cxx_test(
+    kernels_prim_ops_test SOURCES ${_test_srcs}
+)
```
Comment (kirklandsign) [inline kernels/prim_ops/test/CMakeLists.txt:30]: EOF

File: kernels/prim_ops/test/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# @generated by test/utils/generate_gtest_cmakelists.py
```
Comment (kirklandsign) [inline kernels/prim_ops/test/CMakeLists.txt:7]: Remove this line since it's manually written?

File: kernels/prim_ops/test/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+cmake_minimum_required(VERSION 3.19)
+project(prim_ops_test)
+
+# Use C++17 for test.
+set(CMAKE_CXX_STANDARD 17)
+
+set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+
+include(${EXECUTORCH_ROOT}/build/Test.cmake)
+
```
Comment (kirklandsign) [inline kernels/prim_ops/test/CMakeLists.txt:24]: target_link_options_shared_lib(executorch)

File: build/Test.cmake [hunk 4]
```diff
@@ -25,6 +25,7 @@ find_package(executorch CONFIG REQUIRED)
 enable_testing()
 find_package(GTest CONFIG REQUIRED)
 
+target_link_options_shared_lib(executorch)
```
Comment (kirklandsign) [inline build/Test.cmake:28]: Then remove this one

**Pull request merged**